tf2/include/tf2/buffer_core.h

```diff
@@ -103,6 +104,16 @@ class BufferCore
   /** \brief Clear all data */
   void clear();
 
+  /** \brief Clear a specified frame
```

Comment:
a general remark for this function:

* what about "eraseFrame()" instead of "clear()"? It is IMO more inline with std::map's own clear vs. erase functions.
* the doc should explain what it does but, not how it does it.